res/map/AWBWMapConverter.py

```diff
@@ -171,31 +171,45 @@ def indexToTerrainCode(x):
 		149: '10AP',#airport
 		150: '10FC',#base/factory
 		151: '10CT',#city
-		152: '10XX',#Com Tower
+		152: '10TW',#Com Tower
 		153: '10HQ',#HQ
 		154: '10LB',#lab
 		155: '10SP',#port
-		156: '11AP',
+		156: '11AP',#PC
```

Comment:
These numbers are assigned to these teams in AWBW land maybe, but we can assign them as we wish. Not sure it makes sense to label them here.